website/screens/components/file-input/code/examples/formFileInput.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  DxcFileInput,
+  DxcInset,
+  DxcButton,
+  DxcFlex,
+} from "@dxc-technology/halstack-react";
+import { useState } from "react";
+
+const code = `() => {
+  const [files, setFiles] = useState([]);
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    console.log(files.map((file) => file.file.name));
+  };
+
+  const callbackFile = (files) => {
+    setFiles(files);
+  };
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <DxcInset space="2rem">
+        <DxcFlex direction="column" gap="2rem">
+          <DxcFileInput
+            label="Select your files"
+            value={files}
+            callbackFile={callbackFile}
+          />
+          <input type="submit" value="Submit" />
```

Comment:
I think we should use our implementation of a submit button instead of the browser's one: `<DxcButton type="submit" label="Submit" />`.

Reply:
I know it looks very similar to the first example, but I think they will understand the slight but important difference.

Reply:
Agree! Done.